test/utils/readRowsImpl.ts

```diff
@@ -228,7 +231,60 @@ export function readRowsImpl(
     });
 
     let chunksSent = 0;
-    const chunks = generateChunks(serviceParameters);
+    let keyFromRequestClosed: any;
```

Comment:
please do not use `any`s

Reply:
I made changes so that the entire file doesn't contain `any` type. This entire repeated block of if statements has already been replaced with one function call in the next branch/PR anyways.